src/vehicle.cpp

```diff
@@ -4800,6 +4858,17 @@ vehicle_part_range vehicle::get_all_parts() const
     return vehicle_part_range( const_cast<vehicle &>( *this ) );
 }
 
+
+bool vehicle::sentinel_present() const
+{
+    return sentinel_on;
+}
+bool vehicle::need_sentinel() const
+{
+    int orientation = abs( face.dir() ) % 90;
+    return orientation > 29 && orientation < 61 && !sentinel_present();
```

Comment:
should be greater than 14 and less than 76, right?  You can get skew on long vehicles starting on the first turn increment.